security/user/user.go

```diff
@@ -39,3 +39,35 @@ func (su SecurityUser) Fetch(id string, options *types.Options) (types.User, err
 
 	return user, nil
 }
+
+/*
+  Gets the rights of an User using its provided unique id.
+ */
+func (su SecurityUser) GetRights(kuid string, options *types.Options) ([]types.UserRights, error) {
+	if kuid == "" {
+		return []types.UserRights{}, errors.New("Security.User.GetRights: user id required")
+	}
+
+	ch := make(chan types.KuzzleResponse)
+
+	query := types.KuzzleRequest{
+		Controller: "security",
+		Action:     "getUserRights",
+		Id:         kuid,
+	}
+	go su.Kuzzle.Query(query, options, ch)
+
+	res := <-ch
+
+	if res.Error.Message != "" {
+		return []types.UserRights{}, errors.New(res.Error.Message)
```

Comment:
We can't return `nil` without a pointer-type response, which we decided to remove everywhere.
In case of errors, returns will always be empty Entity, empty string, 0 or false depending on return type.
The user should check value of `err` when calling SDK methods, not checking if `res` is `nil`